### PR TITLE
Deal with django 18 deprecation warnings

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: threadedcomments/models.py [hunk 1]
```diff
@@ -9,64 +9,78 @@
 from django.conf import settings
 from django.utils.encoding import force_unicode
 
-DEFAULT_MAX_COMMENT_LENGTH = getattr(settings, 'DEFAULT_MAX_COMMENT_LENGTH', 1000)
+DEFAULT_MAX_COMMENT_LENGTH = getattr(
```
Comment (jlward) [inline threadedcomments/models.py:12]: Almost everything in here is flake8

File: threadedcomments/models.py [hunk 2]
```diff
@@ -143,35 +181,61 @@ class ThreadedComment(models.Model):
...
 
     # Status Fields
-    is_public = models.BooleanField(_('is public'), default = True)
-    is_approved = models.BooleanField(_('is approved'), default = False)
+    is_public = models.BooleanField(_('is public'), default=True)
+    is_approved = models.BooleanField(_('is approved'), default=False)
 
     # Extra Field
-    ip_address = models.IPAddressField(_('IP address'), null=True, blank=True)
+    ip_address = models.GenericIPAddressField(
```
Comment (jlward) [inline threadedcomments/models.py:234]: This was updated

File: threadedcomments/models.py [hunk 3]
```diff
@@ -237,46 +301,72 @@ class Meta:
...
```
Comment (jlward) [inline threadedcomments/models.py:365]: This was updated

File: threadedcomments/models.py (hunk 2 above)
Comment (caffodian) [inline threadedcomments/models.py:234]: i feel like I've asked this before, but this doesn't cause it to think it needs a migration, does it?

File: threadedcomments/models.py [hunk 4]
```diff
@@ -237,46 +301,72 @@ class Meta:
...
+        _('IP address'),
+        null=True,
+        blank=True,
+    )
```
Comment (caffodian) [inline threadedcomments/models.py:369]: same q

File: threadedcomments/models.py (hunk 1 above)
Comment (caffodian) [inline threadedcomments/models.py:12]: so much flake.

File: threadedcomments/models.py (hunk 2 above)
Comment (jlward) [inline threadedcomments/models.py:234]: It will need a migration, which will we handle in PolicyStat.

**Comment by caffodian** [comment]
looks good (unless the IPAddressField -> GenericIPAddressField causes a migration)

**Comment by jlward** [comment]
> looks good (unless the IPAddressField -> GenericIPAddressField causes a migration)

It does, however migrations for this app are handled in PolicyStat. I'll make an issue to update to this version and create the needed migration

**Comment by caffodian** [comment]
Right, because we have the third party migrations directory.  👍

**Pull request merged**